build-system/tasks/bundle-size/index.js

```diff
@@ -53,23 +57,42 @@ const bundleSizeAppBaseUrl = 'https://amp-bundle-size-bot.appspot.com/v0/';
  * @return {Map<string, number>} the bundle size in KB rounded to 2 decimal
  *   points.
  */
-function getBrotliBundleSizes() {
+async function getBrotliBundleSizes() {
   // Brotli compressed size fluctuates because of changes in the RTV number, so
   // normalize this across pull requests by replacing that RTV with a constant.
   const bundleSizes = {};
 
   log(cyan('brotli'), 'bundle sizes are:');
-  for (const filePath of globby.sync(fileGlobs)) {
-    const normalizedFileContents = fs
-      .readFileSync(filePath, 'utf8')
-      .replace(new RegExp(internalRuntimeVersion, 'g'), normalizedRtvNumber);
-
-    const relativeFilePath = path.relative('.', filePath);
-    const bundleSize = parseFloat(
-      (brotliSize.sync(normalizedFileContents) / 1024).toFixed(2)
+  const values = report(process.cwd(), content =>
+    content.replace(
+      new RegExp(internalRuntimeVersion, 'g'),
+      normalizedRtvNumber
+    )
+  );
+  let next = await values.next();
+  let pathMap = undefined;
+  if (!next.done) {
+    const compressed = next.value[0];
+    pathMap = new Map(
+      Array.from(compressed.keys()).map(path => [path, undefined])
```

Comment:
Is it important that paths without bundle size still be present in the map with an undefined value?

Reply:
(or, if it's just to provide the iterable below, could it be `new Set(compressed.keys())` with an initially empty `Map`)

Reply:
That's a good point, I'd planned to use the Map instead of the pre-existing Object.

I'll move this over to a Set.

Reply:
Fixed up, good call.

Originally planned to use the derived Map instead of the pre-exisiting Object. But that would have required changes for all consumers of this method too.